pogobuf/pogobuf.client.js

```diff
@@ -1107,7 +1108,17 @@ function Client(options) {
                             api_url: responseEnvelope.api_url
                         });
 
-                        resolve(self.callRPC(requests, envelope));
+                        signedEnvelope.platform_requests = [];
+                        resolve(self.callRPC(requests, signedEnvelope));
+                        return;
+                    }
+
+                    /* Throttling, retry same request later */
+                    if (responseEnvelope.status_code === 52) {
+                        signedEnvelope.platform_requests = [];
+                        Promise.delay(2000).then(() => {
```

Comment:
Actually the goal of this is not to regenerate a new requestId like before.
In the actual app in case of 52, the request **with the same id** is resend.